config/routes.rb

```diff
@@ -27,6 +27,7 @@
   match '/404', to: 'pages#not_found', via: :all
   match '/500', to: 'pages#internal_server_error', via: :all
   get '/viewers', to: 'pages#viewers', as: 'viewers_page'
+  get '/browserconfig.xml', to: redirect('/browserconfig.xml')
```

Comment:
Is this needed? Wouldn't the `browserconfig.xml` file being in public be enough to serve it at that URL?